usagov/usagov-2021
# Usagov 1155 inconsistent accordion implementation

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1155

## Description

Change the accordion behavior for the following pages:

- https://www.usa.gov/agency-index
- https://www.usa.gov/elected-officials-results?input-street=920+BROAD+ST&input-city=NEWARK&input-state=AK&input-zip=07102#skip-to-h1 

Now only one accordion can be opened at the same time. The previously opened accordion automatically closes after a new one is opened. 

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



1. Go to localhost/agency-index and check that only one accordion can be opened at the same time and the previously opened accordion automatically closes after a new one is opened. 
2. Go to localhost/elected-officials-results?input-street=920+BROAD+ST&input-city=NEWARK&input-state=AK&input-zip=07102#skip-to-h1 and check that only one accordion can be opened at the same time and the previously opened accordion automatically closes after a new one is opened. Check the nested accordions. Here is an example of how it should work: https://www.loom.com/share/041fae626f864b65988cfe2a95ca01e4?sid=8a8fc485-8b92-4ed9-82c2-40618698a32f
3. Now go to the Spanish version of each page and test check if it works as expected. (http://localhost/es/indice-agencias & http://localhost/es/funcionarios-electos-resultados?input-street=920+BROAD+ST&input-city=NEWARK&input-state=AK&input-zip=07102#skip-to-h1).
### Requires New Config

### Requires New Content

### Validation Steps

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
3. Find the commit of this pull request.
4. Build and deploy the changes.
5. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
